transport/x/cherami/example/client.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package example
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/transport/x/cherami"
+	"go.uber.org/yarpc/transport/x/cherami/example/thrift/example/exampleserviceclient"
+)
+
+// This example will create a yarpc client (using cherami transport), and issue an one way rpc call to the server
+// To run this example, the cherami server(https://github.com/uber/cherami-server) needs to be started locally first
```

Comment:
Let's move this test to an integration test using Crossdock/Docker Compose, that way it can run for every commit (and not fail the build).

1. Add Cherami backend and deps to docker-compose.yml
2. Add "cherami" as a transport in AXIS_TRANSPORT_ONEWAY here: https://github.com/yarpc/yarpc-go/blob/dev/docker-compose.yml#L33
3. Instantiate the Cherami transport in the Oneway Crossdock test located at: https://github.com/yarpc/yarpc-go/blob/dev/internal/crossdock/client/oneway/oneway.go

This will ensure that Oneway RPCs can successfully be sent through Cherami.